components/prod/dns_zone.tf

```diff
@@ -52,6 +52,11 @@ resource "azurerm_dns_zone" "hearings" {
   resource_group_name = data.azurerm_resource_group.main.name
 }
 
+resource "azurerm_dns_zone" "vh" {
```

Comment:
it would be better if the zone went in the file: https://github.com/hmcts/azure-public-dns/blob/master/components/prod/vh-hmcts-net.tf

(or the module should create really but smaller change.